prow/getting_started.md

```diff
@@ -0,0 +1,81 @@
+# How to turn up a new cluster
+
+Prow should run anywhere that Kubernetes runs. Here are the steps required to
+set up a basic prow cluster on [GKE](https://cloud.google.com/container-engine/).
+You'll need a GCP project. Prow will work on any Kubernetes cluster, however,
```

Comment:
Link to the cloud console? A lot of people might already have one.

Reply:
Done.